bionic/php71/Dockerfile

```diff
@@ -37,6 +37,7 @@ RUN \
     php7.1-xml \
     php7.1-zip \
     php7.1-mysql \
+    php7.1-phpdbg \
```

Comment:
This appears to install `phpdbg` fine, although I note getting it to run is not trivial. But it appears all those issues can be resolved in the `test-unit.sh` script, rather than the environment here.

@jleckie @jykingston Could one of you please merge this?